<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LieferungForm</class>
 <widget class="QWidget" name="LieferungForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_new">
                <property name="text">
                 <string>&amp;Neu</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_edit">
                <property name="text">
                 <string>&amp;Bearbeiten</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_delete">
                <property name="text">
                 <string>&amp;LÃ¶schen</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_period"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="FilterableTableView" name="tableView_lieferungen"/>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioButton_lieferung">
              <property name="text">
               <string>Lieferung</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_verbrauch">
              <property name="text">
               <string>Verbrauch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_filterArticle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_activateFilter">
              <property name="text">
               <string>&amp;Filter aktivieren</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Details</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView_lieferungenDetails"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterableTableView</class>
   <extends>QTableView</extends>
   <header>lib.FilterableTableView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
